go-controller/pkg/ovn/master.go

```diff
@@ -62,7 +62,7 @@ const (
 //  If true, then either quit or perform a complete reconfiguration of the cluster (recreate switches/routers with new subnet values)
 func (oc *Controller) StartClusterMaster(masterNodeName string) error {
 
-	existingNodes, err := oc.kube.GetNodes()
+	existingNodes, err := oc.watchFactory.GetNodes()
```

Comment:
@abhat I don't think we can change this one either, because I'm not sure we're guaranteed that our informer has the full list the apiserver even though we've waited for cache sync in NewWatchFactory(). I forget the guarantees but I'm dubious. Also this is at startup, so the performance issue is not really as big of a deal.

Reply:
👍